File: Beelina.LIB/Models/UserSetting.cs
using Beelina.LIB.Enums;

namespace Beelina.LIB.Models
{
    public class UserSetting
        : Entity
    {
        public int UserAccountId { get; set; }
        public UserAccount UserAccount { get; set; }
        public bool AllowOrderPayments { get; set; } = false;
        public bool AllowOrderConfirmation { get; set; } = false;
        public bool AllowSendReceipt { get; set; } = false;
        public bool AllowAutoSendReceipt { get; set; } = false;
        public string SendReceiptEmailAddress { get; set; }
        public bool AllowPrintReceipt { get; set; }
        public bool AutoPrintReceipt { get; set; }
        public PrintReceiptFontSizeEnum PrintReceiptFontSize { get; set; } = PrintReceiptFontSizeEnum.Default;
    }
}

File: Beelina.APP/src/app/_models/user-setting.ts
import { Entity } from './entity.model';
import { PrintReceiptFontSizeEnum } from '../_enum/print-receipt-font-size.enum';

export class UserSetting extends Entity {
  allowOrderConfirmation: boolean = true;
  allowOrderPayments: boolean = true;
  allowSendReceipt: boolean = false;
  sendReceiptEmailAddress: string = '';
  allowAutoSendReceipt: boolean = false;
  allowPrintReceipt: boolean = false;
  autoPrintReceipt: boolean = false;
  printReceiptFontSize: PrintReceiptFontSizeEnum = PrintReceiptFontSizeEnum.Default;
}
